sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -146,12 +146,14 @@ private[sql] class SharedState(
    */
   lazy val streamingQueryStatusListener: Option[StreamingQueryStatusListener] = {
     val sqlConf = SQLConf.get
```

Comment:
nit: we can remove this and use `conf` in this class directly.

Reply:
Thanks, then StreamingQueryStatusListener need a little change, done in d763abb.